### ISSUE TITLE
PnP Search WP v4.7.0 : Vertical tab fail to retain ampersand in search box 

### ISSUE THREAD

**Issue by shariq1379** [microsoft-search/pnp-modern-search#2311]
### Check existing issues before logging new ones

Search with ampersand(&) in search box issue fixed, but it fail when click on vertical tab 

**Version used**
Ex: 4.5

**Describe the bug**
We can search with ampersand and search result WP give the correct result, but if fail when click on vertical tab and cut search keyword test after ampersand(&). 

**To Reproduce**
1. Go to search page and search 'T&O'
2. Search result page show the filtered result based on 'T&O' 
3. Click on another vertical tab 
4. It cut '&O' and search box shows 'T' and search result return based on 'T'  

**Screenshots**
<img width="520" alt="PnPSearchIssue2" src="https://user-images.githubusercontent.com/77804584/186264760-4c2137d5-1ac3-4f06-bcdb-62fd155ec7bc.PNG">

<img width="564" alt="PnPSearchIssue3" src="https://user-images.githubusercontent.com/77804584/186264779-a95a6253-892e-4f4d-832c-d0392e34e707.PNG">

**Desktop (please complete the following information):**
 - Browser  chrome, MS Edge

**Comment by wobba** [comment]
Please verify if this is issue is still present with 4.7?

**Comment by shariq1379** [comment]
Yes, this issue still present in v4.7. Actually, It works fine when search 'T&O' in search box, but it fail when click on vertical tab and
search box shows 'T' only.

**Comment by wobba** [comment]
Can you share exact web part setup and connections for repro?

**Comment by shariq1379** [comment]
Yes sure. I have club steps and screenshots in attached document. 
[PnPUpgrade.docx](https://github.com/microsoft-search/pnp-modern-search/files/9417280/PnPUpgrade.docx)
.

**Comment by shariq1379** [comment]
**Search box configuration screenshot**
<img width="340" alt="PnPAmpersandIssue1" src="https://user-images.githubusercontent.com/77804584/186459626-4c763aa2-4ebc-4ab5-a11a-09d9e1fc0362.PNG">

Vertical Web Part configuration screenshot

.../..../sitepages/NewsSearch.aspx?v=news&k={QueryString.k}

<img width="818" alt="PnPAmpersandIssue2" src="https://user-images.githubusercontent.com/77804584/186460076-0b639a7b-4349-4494-8d73-7d43572959b9.PNG">

**Comment by kasperbolarsen** [comment]
I have tested a similar but not identical case using the 4.8.0 version. Rather then using the QueryParameter I just used the most basic setup
One Vertical with 3 tabs ( Word, Excel and PDF )
One Search Box
Three Results web part ( each connected to the Search Box, and one connected to Word, one to Excel and the Last to PDF)

As expected any search for "T&O" works fine across tabs, as the URL remains unchanged.

I then changed the setup to use Query parameters the same way you are, and could reproduce the issue: only the part of the query before the & sign is carried over.
It seems like the & is not URL encoded to %26 as required, however I am not a developer and I can't say if this can be changed in the PnP Modern search source code or not.
